lib/models/user.js

```diff
@@ -238,29 +407,27 @@ User.statics.create = function(email, passwd, callback) {
   // Check to make sure the password is already SHA-256 (or at least is the
   // correct number of bits).
   if (Buffer(passwd, 'hex').length * 8 !== 256) {
-    return callback(
-      new errors.BadRequestError('Password must be hex encoded SHA-256 hash')
-    );
+    return callback(new Error('Password must be hex encoded SHA-256 hash'));
   }
 
-  User.findOne({ _id: email }, function(err, result) {
+  User.findOne({
+    _id: email
+  }, function(err, result) {
     if (err) {
-      return callback(new errors.InternalError(err.message));
```

Comment:
I think these were already fixed in @barbaraliau pull request, but these should remain as the specific error types.